src/experiment_prototype/interface_classes/sequences.py

```diff
@@ -114,6 +114,46 @@ def __init__(
                 )
                 raise ExperimentException(errmsg)
 
+        slice_freqs = []
+        if not self.slice_dict[self.slice_ids[0]].rxonly:
+            for slice_id in self.slice_ids:
+                check_freq = self.slice_dict[slice_id].freq
+                for freq in slice_freqs:
+                    if isinstance(freq, (float, int, complex)):
```

Comment:
If we set `cfs_range` in a slice, the `freq` will default to `None`.  This check probably doesn't need to check float, int and complex, I was just making a quick catch all for any number. When it is not `True` it just skips checking that freq since the condition is only ever `False` when `freq` is `None`